### ISSUE TITLE
The database specified in the config is forced to start with a slash regardless of the server OS

### ISSUE THREAD

**Issue by mariocarro** [Tomas2D/knex-firebird-dialect#371]
This prevents connecting to databases on Windows servers.

**Comment by Tomas2D** [comment]
What's the proposed solution?

**Comment by mariocarro** [comment]
I removed (commented out) the lines:

```typescript
if (customConfig.connection.database[0] !== '/') {
  customConfig.connection.database = `/${customConfig.connection.database}`
}
```

in the constructor or Client_Firebird (lines 31-33 of index.js). That change alone allows the connection to succeed and afterwards everything else works without a hitch.

This change also has the added benefit of allowing the use of database aliases which didn't work before (because everything was treated as a file path).

**Comment by Tomas2D** [comment]
Great. I will update the package later today and let you know.

**Comment by Tomas2D** [comment]
Fixed in version 2.0.3
